app/components/preprint-status-banner/style.scss

```diff
@@ -135,3 +135,16 @@
     }
   }
 }
+@media (max-width: $screen-small-tablet) {
+  .recent-activity {
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
```

Comment:
I'd rather not hide information on small screens. Could it be displayed on the next line instead?
First line: "Accepted", "Modify Decision" button
Second line: [Person] accepted this preprint on [Date]

Reply:
it could be not guaranteed to fit in. Also, some author names might be too long for small screens.

Reply:
I think "not guaranteed to fit" is better than "guaranteed not to fit"

Reply:
And if it were on its own row, it'd be okay to wrap to two lines.

Reply:
it might exceed two lines, do you want in this case to use ellipsis

Reply:
I don't know, I guess it depends how bad it looks. Generally speaking, we shouldn't hide information if we can avoid it.

Reply:
i will give it try